test/Turn-test.js

```diff
@@ -0,0 +1,31 @@
+const chai = require('chai');
+const expect = chai.expect;
+
+const Turn = require('../src/Turn');
+const Card = require('../src/Card')
+
+describe('Turn', function() {
+
+    it('should be a function', function() {
+        const turn = new Turn();
+        expect(Turn).to.be.a('function');
+    })
+
+    it('should be an instance of Turn', function() {
+        const turn = new Turn();
+        expect(turn).to.be.an.instanceOf(Turn);
+    })
+
+    it('should store the user/s guess', function() {
```

Comment:
Nice work realizing that the arguments need to each have their own property, which both require their own test.